package/Shaders/Sky.hlsl

```diff
@@ -205,6 +206,12 @@ PS_OUTPUT main(PS_INPUT input)
 	psout.MotionVectors = screenMotionVector;
 	psout.Normal = float4(0.5, 0.5, 0, 0);
 
+#	if defined(CLOUDS)
```

Comment:
CLOUD_SHADOWS? Or is this a different define?

Reply:
This is a vanilla define.